src/common/latest/analyze-task.ts

```diff
@@ -1,19 +1,21 @@
 import * as tl from "azure-pipelines-task-lib/task";
+import path from "path";
 import {
   JdkVersionSource,
   TASK_MISSING_VARIABLE_ERROR_HINT,
   TaskVariables,
 } from "./helpers/constants";
 import JavaVersionResolver from "./helpers/java-version-resolver";
 import { sanitizeScannerParams, stringifyScannerParams } from "./helpers/utils";
+import { TaskJob } from "./run";
 import { EndpointType } from "./sonarqube/Endpoint";
 import Scanner, { ScannerMode } from "./sonarqube/Scanner";
 
-export default async function analyzeTask(
-  rootPath: string,
-  jdkVersionSource: JdkVersionSource,
-  isSonarCloud: boolean = false,
-) {
+export const analyzeTask: TaskJob = async (endpointType: EndpointType) => {
+  const rootPath = path.join(__dirname, "..");
+  tl.warning(`Root path: ${rootPath}`);
```

Comment:
why are you warning about the root path here?

Reply:
Debugging leftover, I'll drop it thanks